op-guide/backup-restore.md

```diff
@@ -0,0 +1,116 @@
+---
+title: backup and restore
+category: advanced
+---
+
+# Backup and Restore
+
+## About
+
+This document describes how to back up and restore the data of TiDB. Currently, this document only covers full backup and restoration.
+
+Here we assume that the TiDB service information is as follows:
+
+|Name|Address|Port|User|Password|
+|:----:|:-------:|:----:|:----:|:------:|
+|TiDB|127.0.0.1|4000|root|*|
+
+In the process of backing up and restoring data, use the following tools:
+
+- `mydumper`: to export data from TiDB
+- `loader`: to import data into TiDB
+
+### Download TiDB Toolset (Linux)
+
+```bash
+# Download the tool package.
+wget http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz
+wget http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.sha256
+
+# Check the file integrity. If the result is OK, the file is correct.
+sha256sum -c tidb-enterprise-tools-latest-linux-amd64.sha256
+
+# Extract the package.
+tar -xzf tidb-enterprise-tools-latest-linux-amd64.tar.gz
+cd tidb-enterprise-tools-latest-linux-amd64
+```
+
+## Full Backup and Restoration Using `mydumper`/`loader`
+
+The `mydumper` is a powerful data backup tool. See more about [`mydumper`](https://github.com/maxbube/mydumper). 
+
+To back up data, use `mydumper` to export data from TiDB. To restore data, use `loader` to import data into TiDB.
+
+**Note:** Although TiDB supports backing up and restoring data using the official `mysqldump` tool of MySQL, its performance is worse than `mydumper`/`loader`. It will take up much time to back up and restore a large amount of data. Therefore, the tool `mysqldump` is not recommended.
```

Comment:
use following style for a Note:

```
> Note: blah... blah...
```

or 

```
> **Note**： .... blah...
```

Reply:
Change this section to:
You can use `mydumper` to export data from MySQL and `loader` to import the data into TiDB.

> **Note:** Although TiDB also supports the official `mysqldump` tool from MySQL for data migration, it is not recommended to use it. Its performance is much lower than `mydumper` / `loader` and it takes much time to migrate large amounts of data. `mydumper`/`loader` is more powerful. For more information, see [https://github.com/maxbube/mydumper](https://github.com/maxbube/mydumper).